googletest/include/gtest/gtest.h

```diff
@@ -2164,7 +2164,7 @@ constexpr bool StaticAssertTypeEq() noexcept {
 // Define this macro to 1 to omit the definition of TEST(), which
 // is a generic name and clashes with some other libraries.
 #if !GTEST_DONT_DEFINE_TEST
-#define TEST(test_suite_name, test_name) GTEST_TEST(test_suite_name, test_name)
+#define TEST(test_suite_name, test_name) GTEST_TEST(test_suite_name, test_name)  // NOLINT
```

Comment:
The TEST itself is one define, will be expand to one long line. 

Append NOLINT to it to suppress all warnings...